iocage/lib/ioc_list.py

```diff
@@ -151,11 +151,10 @@ def list_all(self, jails):
             if conf["type"] == "template":
                 template = "-"
             else:
-                try:
-                    template = iocage.lib.ioc_common.checkoutput(
-                        ["zfs", "get", "-H", "-o", "value", "origin",
-                         jail_root]).split("/")[3]
-                except IndexError:
+                _origin_property = jail.properties["origin"]
+                if _origin_property and (_origin_property.value != ""):
```

Comment:
() isn't needed ;) along with you can remove the variable `jail_root` now as I think this was the only place it was used. Plus it will shut Travis up!